operator/v1/types_etcd.go

```diff
@@ -43,6 +43,14 @@ type EtcdSpec struct {
 	// +openshift:enable:FeatureGate=HardwareSpeed
 	// +optional
 	HardwareSpeed ControlPlaneHardwareSpeed `json:"controlPlaneHardwareSpeed"`
+
+	// BackendQuotaGiB sets the etcd backend storage size limit in gigabytes. Defaults to 8GiB.
```

Comment:
When did etcd eliminate the 8GB hard limit.  I thought I remembered it being compiled in as: no bigger.

Reply:
iiuc, 8gb is the recommended size, however, the hard limit for the backend db size (i.e. bbolt) is 64gb

Reply:
in fact, bbolt itself is being operated with terabytes, but for etcd case, the limit is 64gb

Reply:
checkout https://github.com/etcd-io/etcd/issues/15354